### PR TITLE
Trying to fix integration test Docker container name conflicts.

### PR DESCRIPTION
**Pull request by helena**

### CODE REVIEW

**Review by chbatey — COMMENTED** [review]
I don't think this is a good idea as we'll leak containers if we don't clean them up

**Comment by helena** [comment]
So no quick fix, just let it fail until we do a cleanup ticket then. It' the container name

File: akka-actor-tests/src/test/scala/akka/io/dns/DockerBindDnsService.scala [hunk 1]
```diff
@@ -76,5 +77,6 @@ trait DockerBindDnsService extends Eventually { self: AkkaSpec ⇒
     self.afterTermination()
     id.foreach(client.killContainer)
     id.foreach(client.removeContainer)
+    client.close()
```
Comment (jrudolph) [inline akka-actor-tests/src/test/scala/akka/io/dns/DockerBindDnsService.scala]: Does that alone help to fix the test?

**Comment by jrudolph** [comment]
I think we have found the real issue, see #26378.

**Comment by helena** [comment]
I was trying to see first what would fix it, without yet doing very much, while working on other things. @chbatey didn't tell me he was doing anything for this also.

**Comment by jrudolph** [comment]
> I was trying to see first what would fix it, without yet doing very much, while working on other things. @chbatey didn't tell me he was doing anything for this also.

Sorry, yes, I decided to finally expedite a fix for this, this morning because it's been failing most of our nightlies for the last week so we had a deeper look into what's going on.

**Pull request closed without merging**